src/exchange_wrappers/OasisV3MatchingExchangeWrapper.ts

```diff
@@ -0,0 +1,44 @@
+import web3Utils from 'web3-utils';
+import BigNumber from 'bignumber.js';
+import {
+  OasisV3MarketOrder,
+} from '../types';
+import { OasisV3MatchingExchangeWrapper as Contract } from '../../migrations/deployed.json';
+
+export class OasisV3MatchingExchangeWrapper {
+  private networkId: number;
+
+  constructor(
+    networkId: number,
+  ) {
+    this.networkId = networkId;
+  }
+
+  public setNetworkId(
+    networkId: number,
+  ): void {
+    this.networkId = networkId;
+  }
+
+  public getAddress(): string {
+    return Contract[this.networkId.toString()].address;
+  }
+
+  public orderToBytes(order: OasisV3MarketOrder): number[] {
+    if (!order.maxPrice) {
+      return [];
```

Comment:
Won't this fail because of https://github.com/dydxprotocol/exchange-wrappers/pull/12/files#diff-a6f8dfc9ce77b0cae15b8e0e23216c07R222 ?

Reply:
See line 214